wolox-books/src/app/screens/auth/screens/sign-up/sign-up.component.ts

```diff
@@ -52,8 +54,10 @@ export class SignUpComponent implements OnInit {
       password_confirmation: this.signUpForm.value.password,
       locale: 'en',
     };
-    console.log(this.user);
-    this.signUpForm.reset();
-    this.router.navigate(['../auth/login'], { relativeTo: this.activatedRoute.parent });
+    this.userService.createUser(this.user).subscribe(($response) => {
```

Comment:
The `$response` variable name it's used more for observables, in this case it's just an object, use `response`.